src/hunspell.mk

```diff
@@ -1,25 +1,25 @@
 # This file is part of MXE. See LICENSE.md for licensing information.
 
 PKG             := hunspell
-$(PKG)_WEBSITE  := https://hunspell.sourceforge.io/
+$(PKG)_WEBSITE  := http://hunspell.github.io/
 $(PKG)_DESCR    := Hunspell
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 1.3.3
-$(PKG)_CHECKSUM := a7b2c0de0e2ce17426821dc1ac8eb115029959b3ada9d80a81739fa19373246c
+$(PKG)_VERSION  := 1.6.1
+$(PKG)_CHECKSUM := 30f593733c50b794016bb03d31fd2a2071e4610c6fa4708e33edad2335102c49
 $(PKG)_SUBDIR   := hunspell-$($(PKG)_VERSION)
 $(PKG)_FILE     := hunspell-$($(PKG)_VERSION).tar.gz
-$(PKG)_URL      := https://$(SOURCEFORGE_MIRROR)/project/hunspell/Hunspell/$($(PKG)_VERSION)/$($(PKG)_FILE)
+$(PKG)_URL      := https://github.com/hunspell/hunspell/archive/v$($(PKG)_VERSION).tar.gz
```

Comment:
Can `$(PKG)_SUBDIR`, `$(PKG)_FILE`, `$(PKG)_URL` and `$(PKG)_UPDATE` be replaced with `$(PKG)_GH_CONF`?